libs/utils/executor.py

```diff
@@ -321,13 +322,34 @@ def _setup_group(self, tc, group):
         # Configure each required attribute
         group.set(**tc['cgroups']['conf'][kind][name])
 
+    def _setup_files(self, tc):
+        if 'files' not in tc:
+            logging.debug('%14s - Configuration not provided', 'Files')
```

Comment:
Since we removed "Files", this will appear as an "Executor" message. Thus, maybe a better log message can be something like: "No files to configure provided"...